src/App.css

```
@@ -103,3 +91,95 @@ ul.specs dt {
 h1 {
   border-bottom: 1px solid gray;
 }
+.shopping-cart {
+  display: -webkit-flex;
+  display: -moz-flex;
+  display: -ms-flex;
+  display: -o-flex;
+  display: flex;
+
+  justify-content: center;
+
+  -webkit-flex-direction: column;
+  -moz-flex-direction: column;
+  -ms-flex-direction: column;
+  -o-flex-direction: column;
+  flex-direction: column;
+
+  box-shadow: 0 0 10px 0px #ff6;
+  border-radius: 10px;
+  padding: 7px;
+}
+.shopping-cart p {
+  text-align: center;
+}
+.shopping-cart__list-item {
+  display: -webkit-flex;
+  display: -moz-flex;
+  display: -ms-flex;
+  display: -o-flex;
+  display: flex;
+
+  justify-content: space-between;
+
+  text-transform: capitalize;
+  padding: 4px 0;
+
+  border-bottom: 1px solid #777;
+
+  list-style: none;
+  margin-left: -20px;
+}
+.gallery {
+  float: left;
+  position: relative;
+  width: 400px;
+  height: 400px;
+  margin-right: 20px;
+  border-radius: 20px;
+  border: 3px outset #977;
+  box-sizing: border-box;
+}
+.gallery--inner {
+  position: absolute;
+  overflow: hidden;
+  width: 394px;
+  height: 394px;
+  border-radius: 20px;
+  background: #aba;
+}
+.gallery-img {
+  background-color: white;
+  width: 400px;
+  height: 100%;
+}
+
+.gallery button {
+  position: absolute;
+  display: inline-block;
+  width: 46px;
+  height: 100%;
+  font-size: 20px;
+  border: none;
+  background: rgba(190, 190, 190, 0.1);
+  text-align: center;
+  z-index: 3;
+}
+.gallery button:focus {
+  outline: none;
+}
+.left {
+  border-bottom-left-radius: 20px;
+  border-top-left-radius: 20px;
+}
+.right {
+  right: 0;
+  border-top-right-radius: 20px;
+  border-bottom-right-radius: 20px;
+}
+.gallery--table {
```

```suggestion
.gallery__table {
```

`table` shouldn't be a modificator. It's rather `element` if you try to follow BEM